Akkadu/rsi-api-widgets
# chore: updated gateway

Please describe the purpose of the pull request.

* [notion page](https://www.notion.so/akkadu/RSI-API-Widget-Implement-Auth-9cf87db1ab2a481ca14ad1a5907ec1df)

## Motivation and Context



## How To Test



* for that demo i used an apiKey demo of our dynamoDb Table 
  * that api key is associated to the roomName : `rgxo` 
* go on the vue preview: https://rsi-akkadu-vue-demo.netlify.app/ it should connect to agora without error.
* if you change the roomname of the url query to something else that : `rgxo` you should get an error: ` An error occured: Your apiKey doesn't have an event with the roomName: rgxof`

## Screenshots (if appropriate)

## Checklist



加油！
